.styleci.yml

```diff
@@ -10,5 +10,4 @@ disabled:
   - phpdoc_no_package
   - phpdoc_summary
   - phpdoc_var_without_name
-  - phpdoc_to_comment
```

Comment:
What does this rule do and why don't we want it?

Reply:
In a previous version of Laravel this was enabled by default, so this was overriding it to disable. But it looks like it's now disabled by default, so we don't need to override!

Reply:
Ah gotcha! And I messed up the double negative there, so hard!